addon/helpers/number.js

```diff
@@ -3,6 +3,15 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([value = 0]) {
+    let num = Number(value);
+    if (`${num}` === 'NaN') {
```

Comment:
For example. Shouldn't this handle the `NaN` case? https://github.com/cowboyd/ember-microstates/blob/master/addon/helpers/number.js#L6-L22